core/primitives-core/Cargo.toml

```diff
@@ -33,6 +34,10 @@ serde_json.workspace = true
 default = []
 protocol_feature_ed25519_verify = []
 
+arbitrary_feature = [
+  "arbitrary",
+  "near-account-id/arbitrary_feature",
+]
```

Comment:
In case the features above were made for consistency with here, since 1.60 you now should be able to have this just be:
```
arbitrary = ["dep:arbitrary", "near-account-id/arbitrary_feature"]
```